orchagent/portsorch.cpp

```diff
@@ -3704,8 +3704,15 @@ void PortsOrch::doLagMemberTask(Consumer &consumer)
 
             if (lag.m_members.find(port_alias) == lag.m_members.end())
             {
-                /* Assert the port doesn't belong to any LAG already */
-                assert(!port.m_lag_id && !port.m_lag_member_id);
+                /* Assert the port is not a LAG */
+                assert(port.m_lag_id == SAI_NULL_OBJECT_ID);
```

Comment:
>assert [](http://example.com/codeflow?start=16&length=6)

`assert` will be nop in release build. This is a runtime eror, we need check and treat it as an error.

Reply:
Right, are you saying this needs to be handled in this PR?

Reply:
yes, please

Reply:
Removed this assert, placed a check for correct port type instead at the begginning.

Reply:
Actually there are lots of asserts, only this is addressed